src/main/scala/strawman/collection/Seq.scala

```diff
@@ -72,6 +72,14 @@ trait IndexedSeqOps[+A, +CC[X] <: IndexedSeq[X], +C] extends Any with SeqOps[A,
     def length: Int = self.length
     def apply(i: Int): A = self(i)
   }
+
+  /** A collection containing the last `n` elements of this collection. */
+  override def takeRight(n: Int): C = fromSpecificIterable(view.takeRight(n))
```

Comment:
Hmm, the implementation inherited from `IterableOps` is this: 
```
  def dropRight(n: Int): C = fromSpecificIterable(View.DropRight(coll, n))
```
which does not seem to make use of the `view` method but instead delegate to `View` directly. If that was changed to:
```
  def dropRight(n: Int): C = fromSpecificIterable(view.dropRight(n))
```  
it would not have to be overridden in `IndexedSeqOps` any more to make use of the overridden `view` method, but I might be misunderstanding the intended inheritance design (if I'm not, all similar methods in `IterableOps` should probably switch from using `View` to `view` to allow sub traits to specialize the view type).